src/main/java/uk/gov/pay/directdebit/webhook/gocardless/services/WebhookGoCardlessService.java

```diff
@@ -58,7 +59,10 @@
     }
 
     public void processEvents(List<GoCardlessEvent> events) {
-        events.forEach(goCardlessService::storeEvent);
+        long start = new Date().getTime();
+        goCardlessService.storeEvents(events);
+        long end = new Date().getTime();
+        LOGGER.error("Time taken to store all events: {} millis", end - start);
```

Comment:
Perhaps this could mention how many events its processed which might be helpful?